geode-core/src/integrationTest/java/org/apache/geode/distributed/internal/membership/gms/fd/GMSHealthMonitorJUnitTest.java

```diff
@@ -450,25 +450,6 @@ public void testRemoveMemberCalledAfterDoingFinalCheckOnCoordinator() throws Exc
     Assert.assertTrue(gmsHealthMonitor.getStats().getSuspectsReceived() > 0);
   }
 
-  /***
-   * validates HealthMonitor.CheckIfAvailable api
-   */
-  @Test
-  public void testCheckIfAvailableNoHeartBeatDontRemoveMember() {
```

Comment:
Is the point of this test to verify that calling `checkIfAvailable(x,x,true)` under the following conditions, returns `false`:

1. no heartbeats have been received from the member
2. we have not yet reached the first timeout time

Notwithstanding the reasoning in the description of this PR, I don't see this situation covered by the remaining tests.

Sure would be nice to be able to use virtual time in this test.

Reply:
The test testFailedCheckIfAvailableRemovesMember verifies that checkIfAvailable(x,x,true) returns false and sets the same pre-conditions that this test used.